src/main/java/seedu/address/logic/parser/DeleteAttributeCommandParser.java

```diff
@@ -12,30 +14,44 @@ public class DeleteAttributeCommandParser implements Parser<DeleteAttributeComma
     /**
      *  Parses the given {@code String} of arguments in the context of the DeleteAttributeCommand
      *  and returns an DeleteAttributeCommand object for execution.
-     *  Command format: delete UUID /attributeName1 /attributeName2 ...
+     *  Command format: delete /UUID /attributeName1 /attributeName2 ...
      *  Examples:
      *  delete
      *  delete /Name /Phone /address
      */
     public DeleteAttributeCommand parse(String args) throws ParseException {
         args = args.trim();
-        String[] parts = args.split("/", -1);
-        String uuid = parts[0].trim();
-        if (parts[0].isEmpty()) {
-            throw new ParseException(Messages.MESSAGE_MISSING_UUID + "\n" + DeleteAttributeCommand.MESSAGE_USAGE);
-        }
+
+        // Split the arguments string on spaces after the first "/"
+        String[] parts = args.split(" ", 2);
+
+        // Validate command structure
         if (parts.length < 2) {
-            throw new ParseException(Messages.MESSAGE_MISSING_ATTRIBUTES
-                    + "\n" + DeleteAttributeCommand.MESSAGE_USAGE);
+            throw new ParseException("Invalid command format. Expected format: "
+                    + "deleteAttribute /UUID /attributeName1 /attributeName2 ...");
```

Comment:
Could this use a constant variable , like was done previously?

Reply:
done